src/client/settings/templates/settings-cache.html

```diff
@@ -14,7 +14,7 @@
 
         <md-input-container>
             <md-checkbox ng-model="settings.config.deleteTmpOnClose">
-                Clear Tmp Folder after closing app?
```

Comment:
this kind of modification, although it makes perfect sense, is also breaking i18n. We currently have most of the strings translated into 50+ langs, and changing it bits by bits will make it hard to maintain, I suggest you to keep String modifications for a bigger, condensed commit, with i18n modifications as well